dvc/commands/experiments/save.py

```diff
@@ -49,7 +49,7 @@ def add_parser(experiments_subparsers, parent_parser):
         "--force",
         action="store_true",
         default=False,
-        help="Save even if hash value for dependencies/outputs changed.",
+        help="Replace experiment if it already exists",
```

Comment:
```suggestion
        help="Replace experiment if it already exists.",
```

Reply:
What about including "save also errors out if nothing changed in the workspace, --force overrides this" ? We can have that in the docs only if you want to keep this short.

Reply:
I'd prefer to keep short, Just in the docs is fine I guess.